src/app/pages/FrontPage/index.test.jsx

```diff
@@ -6,7 +6,7 @@ import { ToggleContextProvider } from '#contexts/ToggleContext';
 import frontPageDataPidgin from '#data/pidgin/frontpage/index-light';
 import pidginMostReadData from '#data/pidgin/mostRead';
 import getInitialData from '#app/routes/home/getInitialData';
-import FrontPageContainer from './index';
+import { FrontPage } from '..';
```

Comment:
If I do that then I won't have the page handler HOCs applied

Reply:
I could do `import { FrontPage } from '#pages';`

Reply:
Do you need to also update it in the `ArticlePage` stories? As we have `import ArticlePage from '.';`?

Reply:
Some of the existing tests have been written for the components without the HOCs applied which is inconsistent but not really a problem. `ArticlePage` is one of the components where it has been tested in isolation without the HOCs applied. If we imported the article page component with HOCs to test then the tests would probably fail with prop type and snapshot errors. Testing a page component in isolation is fine since the `routes/index.js` tests will test all pages with the HOCs applied

Reply:
I hadn't checked if the Article page was tested with or without the HOCs. In that case it make sense 👍